bin/llvm-krun

```diff
@@ -347,7 +347,21 @@ else
 fi
 
 if $dryRun; then
-  if [ -n "$real_output_file" ]; then
+  if [ $binary_input != $binary_output ] && [ -n "$real_output_file" ]; then
+    (
+    if [ -n "$verbose" ]; then
+      set -x
+    fi
+    "$(dirname "$0")/kore-convert" "$expanded_input_file" > "$real_output_file"
+    )
+  elif [ $binary_input != $binary_output ]; then
+    (
+    if [ -n "$verbose" ]; then
+      set -x
+    fi
+    "$(dirname "$0")/kore-convert" "$expanded_input_file"
```

Comment:
```suggestion
    "$(dirname "$0")/kore-convert" -F "$expanded_input_file"
```

If the user specifies binary output but no output file specifically, I presume we do just want to dump binary output to `stdout` here under the assumption that they know what they're doing and will redirect it?